src/data/datetime/unix.rs

```diff
@@ -0,0 +1,18 @@
+use wasm_bindgen::prelude::*;
+use crate::utils::seeder;
+
+#[wasm_bindgen]
+pub fn unix() -> String {
+  let (secs, nanos) = unix_gen();
+  concat_string!(secs.to_string(), nanos.to_string())
+}
```

Comment:
I'd prefer this to have a different js_name than simply unix.

Reply:
unix -> unix_ts